app/src/main/java/org/mozilla/fenix/settings/SecretSettingsFragment.kt

```diff
@@ -39,5 +41,17 @@ class SecretSettingsFragment : PreferenceFragmentCompat() {
             isChecked = context.settings().tabsTrayRewrite
             onPreferenceChangeListener = SharedPreferenceUpdater()
         }
+
+        requirePreference<SwitchPreference>(R.string.pref_key_allow_third_party_root_certs).apply {
+            isVisible = true
+            isChecked = context.settings().allowThirdPartyRootCerts
+            onPreferenceChangeListener = object : SharedPreferenceUpdater() {
+                override fun onPreferenceChange(preference: Preference, newValue: Any?): Boolean {
+                    context.components.core.engine.settings.enterpriseRootsEnabled =
+                        newValue as Boolean
+                    return super.onPreferenceChange(preference, newValue)
+                }
+            }
```

Comment:
I feel you made this as a bit more complicated by not using a `FeatureFlag` and setting that to true to always be displayed. I would advise trying that approach and seeing what you find, but I don't think it's a blocker or needs a re-review. Let me know your thoughts otherwise if you did try it and it did not work out.

Reply:
I started out using it from copying the lines above.  But it felt wrong since this switch is a not turning on or off a feature.  I think it's cleaner this way since I don't need to add a new `FeatureFlags` that will not be use anywhere else.